src/langs/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const NAME_THE_FRAME = {
```

Comment:
```suggestion
export const APP = {
```
I would use a generic name for now.